gm4_note_block_interface/data/gm4_note_block_interface/functions/interface/create.mcfunction

```diff
@@ -1,38 +1,41 @@
+execute if block ~ ~ ~ note_block[note=8] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"D"}'}
+execute if block ~ ~ ~ note_block[note=9] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"D♯"}'}
+execute if block ~ ~ ~ note_block[note=10] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"E"}'}
+execute if block ~ ~ ~ note_block[note=11] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"F"}'}
+execute if block ~ ~ ~ note_block[note=12] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"F♯"}'}
+execute if block ~ ~ ~ note_block[note=13] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"G"}'}
+execute if block ~ ~ ~ note_block[note=14] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"G♯"}'}
+execute if block ~ ~ ~ note_block[note=15] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"A"}'}
+execute if block ~ ~ ~ note_block[note=16] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"A♯"}'}
+execute if block ~ ~ ~ note_block[note=17] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"B"}'}
+execute if block ~ ~ ~ note_block[note=18] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"C"}'}
+execute if block ~ ~ ~ note_block[note=19] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"C♯"}'}
+execute if block ~ ~ ~ note_block[note=20] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"D"}'}
+execute if block ~ ~ ~ note_block[note=21] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"D♯"}'}
+execute if block ~ ~ ~ note_block[note=22] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"E"}'}
+execute if block ~ ~ ~ note_block[note=23] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"F"}'}
+execute if block ~ ~ ~ note_block[note=24] run summon area_effect_cloud ~ ~ ~ {Duration:20,Tags:["gm4_nbi_display"],CustomNameVisible:1,CustomName:'{"text":"F♯"}'}
+
+# updates note block with block under
 execute unless block ~ ~-1 ~ #gm4_note_block_interface:skulls run clone ~ ~-1 ~ ~ ~-1 ~ ~ ~-1 ~ replace force
```

Comment:
I wasn't completely sure if this line is still needed. When I removed it, it still worked fine, so was thinking about removing it.

Reply:
Are those notes possible?

Reply:
yes there's 25 notes, but I mean the clone command

Reply:
This is used to update the note block when backwards tuning. Not sure if it's still needed, cuz it might've been to update it visually for when players have F3 up.

Reply:
It appears this is not necessary, since it works the same if a skull is under the note block. This command is also responsible for the headless pistons thing